src/blog/models.py

```diff
@@ -286,6 +286,19 @@ def like_toggle(self, user):
             self.likes.add(user)
             return True
 
+    @property
+    def liked_by(self):
+        likesArray = []
+        for user in self.likes.all():
+            likesArray.append(
+                {
+                    "username": user.username,
+                    "first_name": user.first_name,
+                    "last_name": user.last_name,
+                }
+            )
```

Comment:
So yeah, I would just use this function here and work with a simple list of strings instead of dictionaries :-)

https://github.com/drikusroor/brazil-blog/blob/d5c3cb36d0ce97c7433588d6f865e9e6e9e4f82e/src/blog/templatetags/user_tags.py#L10-L20